awx/main/dispatch/worker/callback.py

```diff
@@ -48,24 +48,32 @@ def __init__(self):
         self.buff = {}
         self.pid = os.getpid()
         self.redis = redis.Redis.from_url(settings.BROKER_URL)
+        self.redis.client_setname("callbackbrokerworker")
```

Comment:
...and maybe this should match https://github.com/ansible/awx/pull/9461/files#r586609091 for the callback worker process?